pubsub/api/PubsubTest.cs

```diff
@@ -201,53 +203,53 @@ public void DeleteTopic(string topicId, PublisherClient publisher)
             string topicName = PublisherClient.FormatTopicName(_projectId, topicId);
             string subscriptionName =
                 SubscriberClient.FormatSubscriptionName(_projectId, subscriptionId);
-            var delayMs = _retryDelayMs;
+            var delayMs = _retryDelayMs;

```

Comment:
Haven't seen this sample before, but you realize retry is built into the library and can be adjusted with settings?